load-strategies.ts

```diff
@@ -1,6 +1,31 @@
-import { Strategy } from './types/strategy.js';
+import { Strategy, strategySchema } from './types/strategy.js';
+import { readFileSync } from 'fs';
+import config from './config.js';
 
-/**
- * TODO: https://github.com/ConnecMent/dydx-bot/issues/28
- */
-export const loadStrategies = async (): Promise<Strategy[]> => [];
+export const loadStrategies = async (): Promise<Strategy[]> => {
+  const loaded_strategies: Strategy[] = [];
```

Comment:
We won't have multiple strategy files, only one. The file contains multiple strategies as an array. An example would be
```json
[
  {
    "platform": "dydx",
    ...
  },
  {
    "platform": "dydx",
    ...
  }
]
```

Also, use `camelCase` for variable names in JS.